pkg/oc/admin/router/router.go

```diff
@@ -309,15 +334,25 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out, errout io.
 	cmd.Flags().BoolVar(&cfg.StrictSNI, "strict-sni", cfg.StrictSNI, "Use strict-sni bind processing (do not use default cert). Not supported for F5.")
 	cmd.Flags().BoolVar(&cfg.Local, "local", cfg.Local, "If true, do not contact the apiserver")
 
+	cmd.Flags().StringVar(&cfg.MutualTLSAuth, "mutual-tls-auth", cfg.MutualTLSAuth, "Controls access to the router using mutually agreed upon TLS configuration (ala client certificates). You can choose one of 'required', 'optional', or 'none'. The default is none.")
```

Comment:
`(ala client certificates)` ala a typo ?

Reply:
Meaning as an example but I will "rework" that text along with any other new review comments. Thx

Reply:
ala is fine, but slightly obscure -- https://en.wiktionary.org/wiki/ala#Etymology_2

Might be better to say i.e.